src/Dofs/DofHandler.jl

```diff
@@ -12,6 +12,16 @@ Access some grid representation for the dof handler.
 """
 get_grid(dh::AbstractDofHandler)
 
+"""
+    SubDofHandler(dh::AbstractDofHandler, cellset)
+
+Create an `sdh::SubDofHandler` for the cells in `cellset`. All cells in `cellset`
```

Comment:
```suggestion
Create a `sdh::SubDofHandler` for the cells in `cellset`. All cells in `cellset`
```

Just to be sure, are there any invariants on `cellset` which users need to be vary of?

Reply:
Solid reference: https://www.reddit.com/r/grammar/comments/jmrdr/quick_question_an_sd_card_or_a_sd_card/

Good point regarding the set. Currently, it fails at `convert`, e.g. for `UnitRange`, but I'm not sure if it is intended to allow e.g. a range, or different types of sets, in the future. 
So instead of changing the source code, I'll opt to make this a pure doc-pr, and just put `Set{Int}` in the docstring.